config/deploy.rb

```diff
@@ -0,0 +1,27 @@
+# config valid for current version and patch releases of Capistrano
+lock '~> 3.19.1'
+
+set :application, 'search-gov'
+set :repo_url, 'https://github.com/GSA/search-gov'
+set :branch, 'main'
+
+# Set the directory to deploy to
+set :deploy_to, ENV['DEPLOYMENT_PATH']
+
+# Use rbenv to manage Ruby versions
+set :rbenv_type, :user
+set :rbenv_ruby, '3.1.4'
+
```

Comment:
Let's enable puma socket service to start the app upon first request if it is not already running.

```
 set :puma_enable_socket_service, true
 ```
 
 From https://github.com/seuros/capistrano-puma/tree/master?tab=readme-ov-file#systemd-socket-activation